alchemist/build.gradle.kts

```diff
@@ -80,6 +80,11 @@ allprojects {
                 content { groups.forEach { includeGroup(it) } }
             }
         }
+
+        // for tornadofx 2.0.0 snapshot release
+        maven(url = "https://oss.sonatype.org/content/repositories/snapshots")
+        // for LeafletMap
+        maven(url = "https://jitpack.io")
```

Comment:
Argh, both are a problem!
I wrote to @edvin asking for an ETA to tornadofx release.

LeafletMap is unavailable on Central? Can you give me the link to it? If license allows for it, I might republish the library under org.danilopianini.

Reply:
At a minimum, restrict the repos to the sole group using them (see how it's done above)

Reply:
LeafletMap is not on a maven repository as far as I could find. https://github.com/ssaring/sportstracker/tree/master/leafletmap

Reply:
Let me know if I did the restriction correctly